Doc/add faq section (#1129)
pyansys/pyfluent: doc/source/getting_started/installation.rst

If you plan on doing local *development* of PyFluent with Git, install the
latest ``ansys-fluent-core`` package with this code:

.. code:: console

   git clone https://github.com/pyansys/pyfluent.git
   cd pyfluent
   pip install pip -U
   pip install -e .
   python codegen/allapigen.py  # Generates the API files

Any changes that you make locally are reflected in your setup after you restart
the Python kernel.

Launch Fluent
-------------
To launch Fluent from PyFluent, use the :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>`
method:

.. code:: python

  import ansys.fluent.core as pyfluent
  solver = pyfluent.launch_fluent(precision="double", processor_count=2, mode="solver")
  solver.health_check_service.is_serving

To locate the Fluent installation, PyFluent automatically uses the ``AWP_ROOT<ver>``
environment variable, where ``<ver>`` is the three-digit format for the release.
For example, ``AWP_ROOT222`` is the environment variable for the 2022 R2 release. 

On a Windows system, this environment variable is configured when a release is installed.

On a Linux system, you must configure this environment variable to point to the absolute

> path of the installed release. For example, for the 2022 R2 release, you would likely set

Reviewer: This paragraph is about Linux right so the C: path at the end makes no sense.

```suggestion
path of the installed release. For example, for the 2022 R2 release, you might set
```

the ``AWP_ROOT222`` environment variable to point to ``C:\Program Files\ANSYS Inc\v222``.

> To use a non-default installation location, either set the ``AWP_ROOT<ver>`` environment variable

Reviewer: I think we should take this paragraph out.  It was added for the use case where Fluent developers have a local Fluent build area.
